{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -285,7 +297,11 @@
     # https://docs.celeryq.dev/en/stable/userguide/configuration.html#std:setting-timezone
     CELERY_TIMEZONE = TIME_ZONE
 # https://docs.celeryq.dev/en/stable/userguide/configuration.html#std:setting-broker_url
-CELERY_BROKER_URL = env("CELERY_BROKER_URL")
+try:
+    CELERY_BROKER_URL = env("CELERY_BROKER_URL")
+except environ.ImproperlyConfigured:
+    CELERY_BROKER_URL = env("REDIS_URL")
```

Comment:
When I look at this, I wonder why we need `CELERY_BROKER_URL` in the first place 🤔

Reply:
For a project, in the time that Heroku had free redis instances, I used a three instances: a cache backend, celery broker and result backend. But yeah, we can get rid of it since it doesn't add any value.